docs/Developer Notes.md

```diff
@@ -73,3 +73,51 @@ The `make-release` script will then exit. Now, you must go to the Github page
 for the project and make a new release. Set the referenced tag for the release
 to the version you entered into the VERSION file. Add a small write up for
 the new version and attach the resulting tarball to the release.
+
+### Replaying a PCAP with Netflow Data
+
+While any PCAP can be converted into IPFIX records using YAF (see [Generating Data.md](./Generating%20Data.md)), PCAPs which contain raw Netflow v5/ v9/ IPFix
```

Comment:
We should remove the spaces after / in this case since it is being used as an or
https://www.grammarly.com/blog/slash/